app/services/notifications/setup/batch-notifications.service.js

```diff
@@ -6,11 +6,12 @@
  */
 
 const CreateNotificationsService = require('./create-notifications.service.js')
+const NotifyConfig = require('../../../../config/notify.config.js')
 const NotifyEmailService = require('../../notify/notify-email.service.js')
 const NotifyLetterService = require('../../notify/notify-letter.service.js')
 const NotifyUpdatePresenter = require('../../../presenters/notifications/setup/notify-update.presenter.js')
 const ScheduledNotificationsPresenter = require('../../../presenters/notifications/setup/scheduled-notifications.presenter.js')
-const { batchSize } = require('../../../../config/notify.config.js')
+const { setTimeout } = require('node:timers/promises')
```

Comment:
As it's an external package/library it needs to go above our own ([Top of .js files](https://github.com/DEFRA/water-abstraction-team/blob/main/coding_conventions.md#top-of-js-files))